_posts/en/newsletters/2018-07-10-newsletter.md

```diff
@@ -203,10 +203,65 @@ to pursue later." ([source][pwuille comment])
   Bitcoin businesses both internally and hosted on behalf of their
   customers, so some readers of this newsletter may wish to investigate.
 
-- **Building on Bitcoin:** last week was a Bitcoin technology conference
-  well attended by Bitcoin protocol developers.  A [video][bob video]
-  is available, as are several [transcripts][bob transcripts] by Bitcoin
-  developer Bryan Bishop (kanzure).
+## Building on Bitcoin
 
+[**Building on Bitcoin**][bob website] was a Bitcoin technology conference that took
+place in Lisbon last week. It was well attended by both Bitcoin protocol
+developers and applications engineers.  A [video][bob video]
+is available, as are several [transcripts][bob transcripts] by Bitcoin
+developer Bryan Bishop (kanzure).
+
+[bob website]: https://building-on-bitcoin.com/
 [bob video]: https://www.youtube.com/watch?v=XORDEX-RrAI
 [bob transcripts]: http://diyhpl.us/wiki/transcripts/building-on-bitcoin/2018/
+
+The following talks may be of particular interest to Bitcoin Optech
+companies:
+
+- [**Merchant adoption**][bitrefill video] - [Sergej Kotliar][sergej], CEO of
+  Bitrefill gave a personal account of the fee market spike at the end of last
+  year, important UX considerations for Bitcoin and and lightning payments, and
+  Bitrefill's experiences in integrating lightning.  I found this talk
+  fascinating for the real-world empirical data that Sergej shared and his
+  first-hand experience of fees, scaling and lightning.
+
+[bitrefill video]: https://www.youtube.com/watch?v=Cpid31c6HZc&feature=youtu.be&t=8m49s
+[sergej]: https://twitter.com/ziggamon
+
+- [**Designing Lighning Wallets for the Bitcoin Users**][lightning ux video] -
+  [Patrícia Estevão][patricia] gave a talk about UX considerations when
+  extending Bitcoin wallets to support lightning payments.  An interesting
+  talk for any business that is beginning to integrate lightning payments into
+  an existing Bitcoin product.
+
+[lightning ux video]: https://www.youtube.com/watch?v=XORDEX-RrAI&feature=youtu.be&t=6042
+[patricia]: https://twitter.com/patestevao
+
+- [**Blind Signatures in Sciptless Scripts**][blind signatures video] -
+  [Jonas Nick][jonas] spoke about using schnorr signatures as the basis
```

Comment:
I think I used uppercase "Schnorr" in the earlier section about Wuille's proposed BIP, but this is lowercase.  They should probably be consistently capitalized (I don't have a preference).

Reply:
Will capitalize